### ISSUE TITLE
[BUG] NPM7 Fails to install: unmet peer dependency with Cypress 5+ - "could not resolve dependency"

### ISSUE THREAD

**Issue by ryandudek** [IBMa/equal-access#483]
## What tool(s) are you using?

- [ ] `accessibility-checker-engine`
- [ ] `accessibility-checker`
- [ ] `karma-accessibility-checker`
- [ ] `rule-server`
- [ ] `boilerplates`
- [x] `other` - `cypress-accessibility-checker`

## Detailed description

> Node version: 16.5.0
> NPM version: 7.19.1
> Platform: MacOS

> Due to changes in NPMs package lock (lockfileversion 2), an error is thrown on `npm i` when using any version of Cypress that is not explicitly called out as a peer dependency of the `cypress-accessibility-checker` package.

## Steps to reproduce the issue

1. install the latest version of node
2. add `cypress` to your package as a dev dependency (we are on v8.3.0)
3. add `cypress-accessibility-checker` as a dev dependency (we are on v3.1.9)
4. run `npm install`

## Additional information

- our project's package.json:
    ```
    "devDependencies": {
        "@babel/core": "7.14.6",
        "@cypress/webpack-preprocessor": "5.9.1",
        "@mural/ds-manifest-builder": "0.0.0",
        "@storybook/addon-a11y": "6.3.4",
        "@storybook/addon-actions": "6.3.4",
        "@storybook/addon-essentials": "6.3.4",
        "@storybook/addon-links": "6.3.4",
        "@storybook/html": "6.3.4",
        "@storybook/react": "6.3.4",
        "@types/fs-extra": "9.0.12",
        "babel-loader": "8.2.2",
        "cypress": "8.2.0",
        "cypress-accessibility-checker": "3.1.9",
        "rimraf": "3.0.2",
        "ts-loader": "8.3.0"
    },
    "dependencies": {
        "@mural/ds-component-button": "0.0.0",
        "@mural/component-starter": "0.0.0",
        "@mural/ds-config": "0.0.0"
    }
    ```
- error output in terminal:
    ```
    npm ERR! Found: cypress@8.2.0
    npm ERR! node_modules/cypress
    npm ERR!   cypress@"8.2.0" from the root project
    npm ERR!
    npm ERR! Could not resolve dependency:
    npm ERR! peer cypress@"^3 || ^4" from cypress-accessibility-checker@3.1.9
    npm ERR! node_modules/cypress-accessibility-checker
    npm ERR!   cypress-accessibility-checker@"3.1.9" from the root project
    npm ERR!
    npm ERR! Fix the upstream dependency conflict, or retry
    npm ERR! this command with --force, or --legacy-peer-deps
    npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
    npm ERR!
    ```

### Temporary Solution:
The can be fixed by running `npm install --legacy-peer-deps` - but this may require that other peer dependencies need to be added to your package to meet all requirements of other packages.

### Solving this Bug:
This bug can be solved by ensuring that the `peerDependencies` object of the `cypress-accessibility-checker` package is either maintained with new releases of cypress _or_ removed.

**Comment by tombrunet** [comment]
I'll take a look at this this afternoon.

**Comment by ryandudek** [comment]
@tombrunet thanks! It's a pretty straightforward patch, so I created a PR. I'd be happy to discuss the issue via Zoom if you want, my calendar is pretty free today and tomorrow.

If you aren't on npm7 and don't use nvm, testing this one could be less than ideal, so if it's easier to do a screenshare let me know.

**Comment by tombrunet** [comment]
Okay, pretty easy PR. We have a couple of other changes in the pipeline that should wrap up today or tomorrow. Should be out to NPM fairly soon.

**Comment by ryandudek** [comment]
Thanks Tom!

Related, we are using TS in our project, so if you are interested I have some interfaces/types I can share back as well. I have to write up an issue for the equal-access repo still.
